kubeflow/openmpi/README.md

```diff
@@ -79,6 +94,9 @@ Horovod is a distributed training framework for TensorFlow. You may use this pac
 1. Follow the steps in [Quickstart](#quickstart) and use a different entry point.
 
 ```
+# See https://hub.docker.com/r/uber/horovod/tags/ for available tags.
```

Comment:
Let's switch to `uber/horovod:0.13.0-tf1.8.0-torch0.4.0-py2.7`

Reply:
Done.